terraform/cp4ba/variables.tf

```diff
@@ -0,0 +1,232 @@
+}
+//
+//variable "hasEntitlementKey" {
+//  type        = string
+//  description = "Do you have a Cloud Pak for Business Automation Entitlement Registry key (Yes/No, default: No):"
+//}
+
+variable "local_registry_server" {
+  description = "Enter the public image registry or route (e.g., default-route-openshift-image-registry.apps.<hostname>).\nThis is required for docker/podman login validation:"
+}
+
+variable "portworx_is_ready" {
+  type    = any
+  default = null
+}
+
+variable "entitled_registry_key" {
+  type        = string
+//  sensitive = true
+  description = "Do you have a Cloud Pak for Business Automation Entitlement Registry key? If not, Get the entitlement key from https://myibm.ibm.com/products-services/containerlibrary"
+}
+
+variable "namespace" {
+  type        = string
+  description = "namespace for cp4ba"
+}
+
+variable "project_name" {
+  description = "Enter a valid project name. Project name should not be 'openshift' or 'kube' or start with 'openshift' or 'kube'."
+  type = string
+//  validation {
+//    condition = can(regex("^kube+$", var.project_name))
+//    error_message = "Please enter a valid project name that should not be 'openshift' or 'kube' or start with 'openshift' or 'kube'."
+//  }
+}
+
+variable "platform_options" {
+  type        = number
+  description = "Select the cloud platform to deploy. Enter a valid option [1 - 3]: \n 1. RedHat OpenShift Kubernetes Service (ROKS) - Public Cloud \n 2. Openshift Container Platform (OCP) - Private Cloud \n 3. Other ( Certified Kubernetes Cloud Platform / CNCF)"
+}
+
+variable "deployment_type" {
+  type        = number
+  description = "What type of deployment is being performed? Enter a valid option [1 to 2]: \n 1. Demo \n 2. Enterprise"
+}
+
+variable "platform_version" {
+  description = "Enter the platform version"
+}
+
+variable "environment" {
+  default     = "dev"
+  description = "Ignored if `cluster_id` is specified. The environment is combined with `project_name` to name the cluster. The cluster name will be '{project_name}-{environment}-cluster' and all the resources will be tagged with 'env:{environment}'"
+}
+
+variable "use_entitlement" {
+  type = string
+  default = "yes"
+}
+
+variable "public_image_registry" {
+  description = "Have you pushed the images to the local registry using 'loadimages.sh' (CP4BA images)? If not, Please pull the images to the local images to proceed."
+}
+
+variable "local_public_registry_server" {
+  description = "public image registry or route for docker/podman login validation: \n (e.g., default-route-openshift-image-registry.apps.<hostname>). This is required for docker/podman login validation: "
+}
+
+variable "local_registry_user" {
+  description = "Enter the user name for your docker registry: "
+}
+
+variable "local_registry_password" {
+  description = "Enter the password for your docker registry: "
+}
+
+//1
+
+// Flavor will depend on whether classic or vpc
```

Comment:
move all of the `roks` variables together.  Then have another section for all of the `portworx` variables, then a third section for the `cp4ba` specific variables.  This will make more sense to the user.